cypress/integration/page.spec.js

```diff
@@ -1036,7 +1114,7 @@
       page.spyOnCommentsPost();
 
       newCommentForm.nameInput().type('Some name');
-      newCommentForm.bodyArea().type('Some comment body')
+      newCommentForm.bodyArea().type('Some comment body');
```

Comment:
Consider using a more descriptive comment body for the test input to improve readability.